package.json

```diff
@@ -11,6 +11,7 @@
   },
   "main": "cloudinary.js",
   "dependencies": {
+    "cloudinary": "^1.16.0",
```

Comment:
@rebeccapeltz , why is that needed?

Reply:
manual update to latest

Reply:
just noticed that when I merged upstream, it didn't pickup this change - that you had moved cloudinary out of dependencies - not sure why the package.json merge didn't work but I see that you've moved cloudinary to "main" so the npm install runs without error